src/backend/api/reviews.js

```diff
@@ -0,0 +1,71 @@
+const express = require('express');
+const { sum } = require('../database');
+const router = express.Router();
+const knex = require('../database');
+
+// Returns all reviews
+// GET api/reviews/
+router.get('/', async(request, response) => {
+    try {
+        const getAllReviews = await knex('review'); //.select('title');
```

Comment:
Nitpick(naming): Try to name your variables as nouns, "allReviews" or "reviews".
Reserve verb names for functions!

Reply:
i renamed it . thank you  so much. i will keep that in mind in future